src/lib/mongoBackend/mongoCreateSubscription.h

```diff
@@ -48,7 +48,6 @@ extern std::string mongoCreateSubscription
 (
   const ngsiv2::Subscription&          sub,
   OrionError*                          oe,
-  std::map<std::string, std::string>&  uriParams,
```

Comment:
In the past we opted for a "have the method signature homogeneous, even when some parameters could not be used in some cases". However, after the experience, I now think that is better to adopt a "each function uses only what it needs" to improve function/method encapsulation. Thus, this change.

NTC (just informative).